### PR TITLE
remove unique constraint from liquibase indexes for name

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/resources/liquibase/db-changelog-1.0.xml [hunk 1]
```diff
@@ -127,10 +127,11 @@
             </not>
         </preConditions>
         <createIndex indexName="p_name_wspace_wspace_type"
-                     tableName="project" unique="true">
+                     tableName="project">
             <column name="name" type="varchar(255)"/>
             <column name="workspace" type="varchar(255)"/>
             <column name="workspace_type" type="int"/>
+            <column name="deleted" type="boolean"/>
```
Comment (ravishetye) [inline backend/src/main/resources/liquibase/db-changelog-1.0.xml:155]: why is this change required?

**Review by ravishetye — CHANGES_REQUESTED** [review]
we will have to drop and recreate the index, since this will now modify the index already created.
we will have to do it in a new changeset

**Comment by anandjakhaniya** [comment]
Updated indexes as per review comment

**Comment by ravishetye** [comment]
can you pull in master, I do not expect so many changes

**Review by ravishetye — APPROVED** [review]

**Pull request merged**